CITATION.cff

```diff
@@ -0,0 +1,41 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Dynamic Modeling of Health Impact Assessment in R
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Hendriek
+    family-names: Boshuizen
+    email: hendriek.boshuizen@rivm.nl
+  - given-names: Malte
+    family-names: Lüken
+    email: m.luken@esciencecenter.nl
+  - given-names: Flavio
+    family-names: Hafner
+    email: f.hafner@esciencecenter.nl
+  - given-names: Laurens
+    family-names: Bogaardt
+    email: laurens.bogaardt@rivm.nl
+repository-code: 'https://github.com/Dynamo-HIA/hiaR'
+url: 'https://dynamo-hia.github.io/hiaR/'
+abstract: >-
```

Comment:
I don't have a preference here, but perhaps a more readable alternative would be to use "them" instead of him/her: "import their data ... that are of interest to them."

Reply:
Fixed this!